rust/db_handling/src/identities.rs

```diff
@@ -1294,17 +1294,21 @@ fn prepare_secret_key_for_export(
 /// coloration, so that the difference with safe QR codes is immediately visible
 /// on screen.
 #[cfg(feature = "signer")]
-pub fn export_secret_key(
-    database_name: &str,
+pub fn export_secret_key<P>(
+    db_path: P,
     multisigner: &MultiSigner,
     expected_seed_name: &str,
-    network_specs_key: &NetworkSpecsKey,
+    network_specs_key_hex: &str,
     seed_phrase: &str,
-    pwd: Option<&str>,
-) -> Result<MKeyDetails> {
-    let network_specs = get_network_specs(database_name, network_specs_key)?;
+    mut pwd: Option<String>,
```

Comment:
would be nice to add simple comment  to which parameters it is. Like what should we pass to pwd.

Reply:
I try to make arg names self-explanatory, so it's not nessesary add any comments. You're right, `pwd` is not quite clear name, I can rename it to `key_password` or `secret_password`